erpnext/assets/doctype/asset/test_asset.py

```diff
@@ -221,7 +221,7 @@ def test_scrap_asset(self):
 			asset.precision("gross_purchase_amount"),
 		)
 		pro_rata_amount, _, _ = asset.get_pro_rata_amt(
-			asset.finance_books[0], 9000, add_months(get_last_day(purchase_date), 1), date
+			asset.finance_books[0], 9000, get_last_day(add_months(purchase_date, 1)), date
```

Comment:
The previous way worked only for months with 30 days and February, not for months with 31 days.